admin_guide/overcommit.adoc

```diff
@@ -42,6 +42,39 @@ the specified resource as is available with the lowest local priority. In low
 resource situations, containers that specify no resource requests are given the
 lowest quality of service.
 
+[[tune-buffer-chunk-limit]]
+=== Tune Buffer Chunk Limit
+
+If Fluentd logger is unable to keep up with a high number of logs, it will need
+to switch to file buffering to reduce memory usage and prevent data loss.
+
+The Fluentd `buffer_chunk_limit` is determined by the environment variable
+`buffer_size_limit`. The file buffer size per output is determined by the
+environment variable `file_buffer_limit`. The permanent volume size has to be
```

Comment:
the environment variable 'FILE_BUFFER_LIMIT'.
The default value of FILE_BUFFER_LIMIT is 2g.